test/test_cuda.py

```diff
@@ -4884,6 +4884,27 @@ def test_no_triton_on_import(self):
 
 @unittest.skipIf(not TEST_CUDA, "CUDA not available, skipping tests")
 class TestMemPool(TestCase):
+    def _setup_mempool_limited_memory_test(self, additional_allowed_memory_in_mb):
```

Comment:
I'm wondering if this needs a `serialTest` decorator as exactly tracking allocation stats seems scary if we allow parallel execution of tests